stdlib/3/dbm/dumb.pyi

```diff
@@ -0,0 +1,31 @@
+
+import sys
+from typing import Union, MutableMapping, Iterator, Optional, Type
+from types import TracebackType
+
+if sys.version_info >= (3, 8):
+    from typing import Final
+else:
+    from typing_extensions import Final
+
+_KeyType = Union[str, bytes]
+_ValueType = Union[str, bytes]
+
+error: Final[Type[OSError]]
+
+class _Database(MutableMapping[_KeyType, _ValueType]):
+
+    def __init__(self, filebasename: str, mode: str, flag: str = ...) -> None: ...
+    def sync(self) -> None: ...
+    def iterkeys(self) -> Iterator[_KeyType]: ...  # undocumented
+    def close(self) -> None: ...
+    def __getitem__(self, item: _KeyType) -> _ValueType: ...
```

Comment:
The argument is called `key`, not `item`.

Reply:
And it looks like the return type is always `bytes`, because it gets read from something opened with `rb`.